### PR TITLE
Fixed dropdown icon

### LINKED ISSUE #2199: Dropdown icon in Server Settings is black in Dark theme
#### Software and hardware versions 
Current master branch

#### Problem description
Dropdown icon in Server Settings is black in Dark theme

screenshot_2018-05-14-11-49-06

### PR DESCRIPTION
**Pull request by grzesiek2010**
Closes #2199 

#### What has been done to verify that this works as intended?
I tested new appearance in Server Settings

#### Why is this the best possible solution? Were any other approaches considered?
I added a vector file instead of pngs and fixed the color.

#### Are there any risks to merging this code? If so, what are they?
No.

#### Do we need any specific form for testing your changes? If so, please attach one.
No.

#### Does this change require updates to documentation? If so, please file an issue [here]( https://github.com/opendatakit/docs/issues/new) and include the link below.
No.

#### Before submitting this PR, please make sure you have:
- [x] run `./gradlew pmd checkstyle lint findbugs` and confirmed all checks still pass.
- [x] verified that any new UI elements use theme colors so that they work with both light and dark themes.

### CODE REVIEW

**Comment by mmarciniak90** [comment]
@grzesiek2010 dropdown icon looks good on Android 5.1, 6.0, 7.0, 8.1. 
Unfortunately, Collect crashes when I try to go to `Server` options on Android 4.4, 4.2, 4.1.

@opendatakit-bot unlabel "needs testing"

File: collect_app/src/main/res/drawable/ic_arrow_drop_down.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<vector android:height="36dp" android:viewportHeight="24.0"
+    android:viewportWidth="24.0" android:width="36dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="?attr/colorControlNormal" android:pathData="M7,10l5,5 5,-5z"/>
```
Comment (shobhitagarwal1612) [inline collect_app/src/main/res/drawable/ic_arrow_drop_down.xml]: Please use `?iconColor`

**Review by shobhitagarwal1612 — DISMISSED** [review]

**Comment by grzesiek2010** [comment]
@opendatakit-bot label "needs testing"

**Comment by grzesiek2010** [comment]
@mmarciniak90 I fixed the issue please continue testing.

**Comment by mmarciniak90** [comment]
**Tested with success!**

Verified on Android: 4.1, 4.2, 4.4, 6.0, 7.0, 8.1

@opendatakit-bot unlabel "needs testing"
@opendatakit-bot label "behavior verified"

**Review by lognaturel — APPROVED** [review]

**Pull request merged**